src/components/PositionCard/StablePositionCard/index.tsx

```diff
@@ -118,16 +118,16 @@ export default function FullStablePositionCard({ poolName, border }: StablePosit
 
   const [stablePoolData, userData] = useStablePoolsData(poolName)
   const { name, tokens } = stablePoolData
-  const { address: poolAddress } = STABLESWAP_POOLS[poolName]
+  const { address: poolAddress, poolTokens: nonTlpTokens } = STABLESWAP_POOLS[poolName]
   const poolTokens = tokens.map(({ token }) => token)
-  const [token0, token1, token2] = poolTokens
-  const [currency0, currency1, currency2] = poolTokens.map(token => unwrappedToken(token))
+
+  const currencies = nonTlpTokens.map(token => unwrappedToken(token))
   const [showMore, setShowMore] = useState(false)
 
-  const backgroundColor1 = useColorWithDefault('#2172E5', token0)
-  const backgroundColor2 = useColorWithDefault('#2172E5', token1)
+  const backgroundColor1 = useColorWithDefault('#2172E5', nonTlpTokens[0])
+  const backgroundColor2 = useColorWithDefault('#2172E5', nonTlpTokens[nonTlpTokens.length - 1])
   // @TODO Update styling so it can handle 3 color gradient
-  const backgroundColor3 = useColorWithDefault('#2172E5', token2)
+  // const backgroundColor3 = useColorWithDefault('#2172E5', token2)
```

Comment:
This was added as a reminder to incorporate `n` token colors to the background gradient.  I'll create a card for this

Please don't add commented code to prod
```suggestion
```